install/mysql/glpi-9.2-empty.sql

```diff
@@ -7040,6 +7040,7 @@ CREATE TABLE `glpi_taskcategories` (
   `entities_id` int(11) NOT NULL DEFAULT '0',
   `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
   `taskcategories_id` int(11) NOT NULL DEFAULT '0',
+  `knowbaseitemcategories_id` int(11) NOT NULL DEFAULT '0',
```

Comment:
I do not understand why a field is added here; or maybe PR description is no correct.